apps/condo/package.json

```diff
@@ -24,7 +24,8 @@
     "lint-i18n-translations": "node ./../../bin/lint-i18n-translations.js",
     "refresh-sbbol-tokens": "node ./../../bin/sbbol.js",
     "create-integration": "node ./../../bin/create-integration.js",
-    "sbbol:crypto:get": "node ./../../bin/sbbol/crypto.js"
+    "sbbol:crypto:get": "node ./../../bin/sbbol/crypto.js",
+    "subscription:prolong": "node ../../bin/prolong-subscription.js"
```

Comment:
what do you think about domain folders?

Reply:
Initially, I have done it for sbbol, but due to small amount of scripts, I've reverted that approach to avoid having most folders with one file.

Reply:
Btw, scripts can be moved from `bin` to `domains/…/scripts`!
It seems like a good idea.

Reply:
But, it depends… For example SBBOL features covering many domains, — authentication belongs to `user` domain, organization syncing belongs to `organisation` domain, subscriptions etc.

Reply:
I think, to avoid complexity, — to leave things as is.
When we will have tens scripts, — some sort of organising will be actual.